src/app/(client)/meal/detail/components/meal-detail-section.tsx

```
@@ -70,16 +81,18 @@ const MealDetailSection = ({ meal }: MealDetailSectionProps) => {
       });
       alert('식사 정보가 수정되었습니다.');
     } catch (error) {
-      alert('식사 정보 수정에 실패했습니다.');
-      console.error('Failed to update meal:', error);
+      const isKnownError = typeof error === 'object' && error !== null && 'title' in error && 'description' in error;
+      error = isKnownError ? error : ERROR_MESSAGES.MEAL_POST_FAILED;
```

Biome (1.9.4)

[error] Reassigning a catch parameter is confusing.

The catch parameter is declared here:

Use a local variable instead.

(lint/suspicious/noCatchAssign)